src/preprocessor/helper/compatStyle.ts

```diff
@@ -307,6 +307,11 @@ export default function globalCompatStyle(option: any, isTheme?: boolean) {
                 deprecateLog('nameGap property in radar component has been changed to axisNameGap');
             }
         }
+        each(radarOpt.indicator, function (indicatorOpt: any) {
+            if (__DEV__ && indicatorOpt.text) {
```

Comment:
There may be an unnecessary empty function invoking as `__DEV__` check is put into the loop.

The final output will be as follows, (minimized)

```js
Jp(t.indicator,(function(t){0}))
```